app/src/main/res/layout/activity_main.xml

```diff
@@ -4,26 +4,97 @@
+        android:background="@drawable/button_border"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/all_toast_btn"
+        android:textAlignment="center"
+        android:textAppearance="@style/TextAppearance.AppCompat.Headline"
+        android:textColor="#2196F3"
+        android:typeface="normal"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <Button
-        android:id="@+id/button"
-        android:layout_width="135dp"
-        android:layout_height="85dp"
-        android:text="Button"
-        app:layout_constraintBottom_toTopOf="@+id/textView"
+
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/android_ui_btn"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="25dp"
+        android:layout_marginRight="30dp"
+        android:background="@drawable/button_border"
+        android:fontFamily="sans-serif-condensed-medium"
+        android:foreground="?android:attr/selectableItemBackground"
+        android:text="@string/android_btn"
+        android:textAppearance="@style/TextAppearance.AppCompat.Headline"
+        android:textColor="@color/text_color_mainActivity"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@id/button" />
+
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/different_layout_activity_btn"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="25dp"
+        android:layout_marginRight="30dp"
+        android:background="@drawable/button_border"
+        android:fontFamily="sans-serif-condensed-medium"
+        android:foreground="?android:attr/selectableItemBackground"
+        android:text="@string/different_layout_btn"
+        android:textAppearance="@style/TextAppearance.AppCompat.Headline"
+        android:textColor="@color/text_color_mainActivity"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/android_ui_btn" />
+
+    <ScrollView
+        android:layout_width="328dp"
+        android:layout_height="55dp"
+        android:layout_marginTop="8dp"
+        android:fillViewport="false"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.265"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:ignore="MissingConstraints">
+
+        <LinearLayout
```

Comment:
What is the use of this layout?

Reply:
It is because I was trying with scroll view and Linear Layout Implementation